Userland/Libraries/LibPDF/ObjectDerivatives.h

```diff
@@ -207,6 +208,27 @@ class IndirectValue final : public Object {
     Value m_value;
 };
 
+class BitmapObject final : public Object {
```

Comment:
I don't love having to add a new type of Object, but I couldn't think of a much nicer way to do this.